### DISCUSSION TITLE
FluentUI Blazor issue - When is the next release ?

### DISCUSSION THREAD

**Discussion by AClerbois** [microsoft/aspire#1120] [Q&A] [answered]
Hi beautiful mainteners 😃 , 

I get some error with the current package nuget using an older version of FluentUI Blazor. 

Good news, it seems you fixed but I need a new version of the Aspire package NuGet. 

Ok, I'm lazy, I don't want to create a package manualy. 

Quick question : What is the plan for the new (pre)release of this awesome project ? 

Br,

Adri

**Comment by DamianEdwards** [comment, marked as answer]
Next preview will be out in early December.

**Reply by AClerbois** [reply]
Thank @DamianEdwards
